spec/dummy/db/migrate/20150909133655_add_previous_refresh_token_to_access_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreviousRefreshTokenToAccessTokens < ActiveRecord::Migration
+  def change
+    add_column :oauth_access_tokens, :previous_refresh_token, :string, default: "", null: false
+  end
+end
```

Comment:
Do you want to update this migration with the one with the new coding style? Thanks!